index.html

```diff
@@ -41,43 +48,208 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
       <div class="socials-section">
         <button class="connect-link padding">LET'S CONNECT</button>
```

Comment:
- [ ] As said previously by the reviewer, Kindly make sure you use a `heading tag` here instead of a button to make your markup more semantic.
Hint: `You may consider using a h3,h4, h5 or h6`